media-plugins/distrho-ports/metadata.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
+    <email>webmaster@gentoostudio.org</email>
```

Comment:
This could use a `<name>` element ("Gentoo Studio Project" or similar? whatever you feel is appropriate here)